_data/import-public.rb

```diff
@@ -9,12 +9,60 @@
 require 'json'
 
 DATA_DIR = File.dirname __FILE__
+TEAM_DIR = "_team"
 TEAM_API_BASE_URL = 'https://team-api.18f.gov/public/api/'
+TEAM_ENDPOINT = 'team'
+MEMBER_PROPERTIES = [
```

Comment:
You can define an Array of string constants using the `%w(foo bar baz)` style. (Which I learned from [Rubocop](http://batsov.com/rubocop/) on other projects.)